src/main/java/model/Entities/Return.java

```diff
@@ -11,9 +11,9 @@
 
     public int rid;
     public Timestamp returnDateTime;
-    public int endOdometer;
+    public int endOdometer; // TODO: Change to long
```

Comment:
Long might be too big, int is enough because I don't think that cars normally can drive more than 2,147,483,647 km or miles